static/js/components/VideoCard_test.js

```diff
@@ -58,6 +60,14 @@ describe('VideoCard', () => {
     sinon.assert.called(showShareDialogStub);
   });
 
+  it(`should have a title that links to the video detail page`, () => {
+    let wrapper = renderComponent();
+    let title = wrapper.find(".video-card-body h4");
+    assert.isTrue(title.exists());
+    assert.equal(title.text(), video.title);
+    assert.isTrue(title.find('a').exists());
```

Comment:
Might be good to have an additional assert that checks if the link URL is correct.